### ISSUE TITLE
代码理解

### ISSUE THREAD

**Issue by jerryniu0624** [dengweihuan/SSDGL#7]
请问代码中哪里体现了the weighted softmax with cross entropy loss is introduced to reweight each class probability。我在运行SSDGL_1_0_pavia_small的时候，感觉和FPGA没有啥区别

**Comment by jerryniu0624** [comment]
Also, is the alpha : the number of stratified training data in your paper set 20 in your code?

**Comment by jerryniu0624** [comment]
Thx for your work, where is the compression factor mentioned in "The pseudocode of SSDGL" in your code? I am a bit confused
